src/backend/commands/createas.c

```diff
@@ -1335,7 +1373,12 @@ check_ivm_restriction_walker(Node *node, check_ivm_restriction_context *ctx, int
 							(errcode(ERRCODE_FEATURE_NOT_SUPPORTED),
 							 errmsg("this query is not allowed on incrementally maintainable materialized view"),
 							 errhint("subquery with outer join is not supported")));
+
+				ctx->has_exists_subquery = true;
```

Comment:
This seems a flag to indicate that "we are now checking EXISTS subquery", so "in_exists_subquery" is proper than has_exists_subquery, isn't it?